diagnostic.md

```diff
@@ -23,7 +29,7 @@ Write the jQuery code to navigate from one of the list item (`li`) DOM nodes to
 the `form` DOM node. (Assume the list item has a class of "listerItem").
 
 <!-- your answer starts here -->
-
+$('body').find('form')
```

Comment:
So here we were seeing if you could start at the `li` node and traverse up to find the form. Something like this:

`$('li').parent().parent().find('form')`